app/client/debug.go

```diff
@@ -0,0 +1,9 @@
+//go:build debug
```

Comment:
I was about to suggest:

```suggestion
//go:build debug
// +build debug
```

but then I realised that it seems that the `+build ...` syntax is obsolete since Go 1.17

Nice